### PR TITLE
Update resource names

### PR DESCRIPTION
**Pull request by victorhurdugaci**
React to DNX resource name changes.

Please review @NTaylorMullen or @Tratcher

### CODE REVIEW

File: test/Microsoft.Framework.WebEncoders.Tests/UnicodeHelpersTests.cs [hunk 1]
```diff
@@ -157,7 +157,10 @@ private static bool[] ReadListOfDefinedCharacters()
             HashSet<string> seenCategories = new HashSet<string>();
 
             bool[] retVal = new bool[0x10000];
-            string[] allLines = new StreamReader(typeof(UnicodeHelpersTests).GetTypeInfo().Assembly.GetManifestResourceStream("../../unicode/UnicodeData.txt")).ReadAllLines();
+
+            var assembly = typeof(UnicodeHelpersTests).GetTypeInfo().Assembly;
+            var resourceName = assembly.GetName().Name + "." + "UnicodeData.txt";
```
Comment (Tratcher) [inline test/Microsoft.Framework.WebEncoders.Tests/UnicodeHelpersTests.cs]: ` + "." + "UnicodeData.txt"` is redundant, merge them.

Comment (NTaylorMullen) [inline test/Microsoft.Framework.WebEncoders.Tests/UnicodeHelpersTests.cs]: `+ ".UnicodeData.txt"`

Comment (victorhurdugaci) [inline test/Microsoft.Framework.WebEncoders.Tests/UnicodeHelpersTests.cs]: (facepalm) I've changed so many strings today that I don't notice this anymore

**Comment by Tratcher** [comment]
:shipit:

**Comment by NTaylorMullen** [comment]
:shipit:

**Pull request merged**